config/development.json

```diff
@@ -28,7 +28,9 @@
 	"features": {
 		"ad-tracking": false,
 		"build/sites-dashboard": true,
+		"calypso/help-center": true,
 		"calypsoify/plugins": true,
+		"checkout/help-center": true,
```

Comment:
Do we need this anymore? If enabled in all Calypso, let's delete the arbitrary distinction for checkout.

Reply:
Thanks Omar, it makes sense and it will simplify things 👌